palmsy/iqm.cfg

```diff
@@ -2,19 +2,21 @@
 
 iqmload mesh.iqm
 
-iqmskin leaves leaves.png
-iqmcullface leaves 0
-iqmalphatest leaves 0.2
+iqmskin leafs leafs leafs_mask
+iqmcullface leafs 0
+iqmalphatest leafs 0.2
+iqmmaterial leafs 1
 
-iqmskin trunk trunk.jpg
-iqmbumpmap trunk trunk_n.jpg
-iqmcolor leaves -1
+iqmskin trunk trunk
+iqmbumpmap trunk trunk_n
+iqmmaterial trunk 0
 
 iqmanim "mapmodel" "idle.iqm" 8
-iqmpitch * 1
+iqmpitch "basebone" 1
 
-mdlscale 100
+mdlscale 50
 mdlspec 10
 mdlbb 6 80
 mdlellipsecollide 1
+mdlcolor -1
 mdlyaw 90
```

Comment:
No newline at end of file.

Reply:
done, I should be paying more attention to these ending newlines

Reply:
done, I should be paying more attention to these ending newlines